src/main/java/de/themorpheus/edu/taskservice/database/repository/VotingRepository.java

```diff
@@ -0,0 +1,9 @@
+package de.themorpheus.edu.taskservice.database.repository;
+
+import de.themorpheus.edu.taskservice.database.model.VotingModel;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface VotingRepository extends JpaRepository<VotingModel, Integer> {
```

Comment:
Where's the VotingModel?

Reply:
The model was added with issue 128. It is already in the experimental branch.